src/java/org/apache/cassandra/locator/ReplicaPlan.java

```diff
@@ -106,18 +106,25 @@ ForTokenRead withContact(EndpointsForToken newContact)
     public static class ForRangeRead extends ForRead<EndpointsForRange>
     {
         final AbstractBounds<PartitionPosition> range;
+        final int rangeCount;
 
-        public ForRangeRead(Keyspace keyspace, ConsistencyLevel consistencyLevel, AbstractBounds<PartitionPosition> range, EndpointsForRange candidates, EndpointsForRange contact)
+        public ForRangeRead(Keyspace keyspace, ConsistencyLevel consistencyLevel, AbstractBounds<PartitionPosition> range, EndpointsForRange candidates, EndpointsForRange contact, int rangeCount)
```

Comment:
Nit: We could break this line
```suggestion
        public ForRangeRead(Keyspace keyspace,
                            ConsistencyLevel consistencyLevel,
                            AbstractBounds<PartitionPosition> range,
                            EndpointsForRange candidates,
                            EndpointsForRange contact,
                            int rangeCount)
```

Reply:
+1